makefiles/common.mk

```diff
@@ -176,17 +178,17 @@ THEOS_TARGET_LIBRARY_PATH := $(THEOS_LIBRARY_PATH)/$(THEOS_TARGET_NAME)
 _THEOS_PACKAGE_FORMAT := $(or $(call __schema_var_last,,$(_THEOS_TARGET_NAME_DEFINE)_PACKAGE_FORMAT),$(call __schema_var_last,,PACKAGE_FORMAT),$(_THEOS_TARGET_DEFAULT_PACKAGE_FORMAT),none)
 _THEOS_PACKAGE_LAST_FILENAME = $(call __simplify,_THEOS_PACKAGE_LAST_FILENAME,$(shell cat "$(_THEOS_LOCAL_DATA_DIR)/last_package" 2>/dev/null))
 
-_THEOS_INTERNAL_SEARCHPATHS :=
+_THEOS_INTERNAL_SEARCHPATHS := $(TARGET_PRIVATE_FRAMEWORK_PATH)
 ifeq ($(THEOS_PACKAGE_SCHEME),)
-	_THEOS_INTERNAL_SEARCHPATHS += \
-		$(THEOS_VENDOR_LIBRARY_PATH) \
-		$(THEOS_LIBRARY_PATH) \
-		$(THEOS_TARGET_LIBRARY_PATH)
-endif
 _THEOS_INTERNAL_SEARCHPATHS += \
-	$(TARGET_PRIVATE_FRAMEWORK_PATH) \
-	$(THEOS_VENDOR_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(or $(THEOS_PACKAGE_SCHEME),rootful) \
-	$(THEOS_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(or $(THEOS_PACKAGE_SCHEME),rootful)
+	$(THEOS_VENDOR_LIBRARY_PATH) \
+	$(THEOS_LIBRARY_PATH) \
+	$(THEOS_TARGET_LIBRARY_PATH)
+else
+_THEOS_INTERNAL_SEARCHPATHS += \
+	$(THEOS_VENDOR_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(THEOS_PACKAGE_SCHEME) \
+	$(THEOS_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(THEOS_PACKAGE_SCHEME)
+endif
```

Comment:
I'm not convinced this conditional works in the same way as other changes with `$(or $(THEOS_PACKAGE_SCHEME),rootful)` -> `$(THEOS_PACKAGE_SCHEME)`

Reply:
Documenting the values of `_THEOS_INTERNAL_SEARCHPATHS` after all the conditionals here for clarity.

(previously discussed else-where)

Previously:

```make
ifeq ($(THEOS_PACKAGE_SCHEME),)
    _THEOS_INTERNAL_SEARCHPATHS = \
        $(THEOS_VENDOR_LIBRARY_PATH) \
        $(THEOS_LIBRARY_PATH) \
        $(THEOS_TARGET_LIBRARY_PATH) \
        $(TARGET_PRIVATE_FRAMEWORK_PATH) \
        $(THEOS_VENDOR_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/rootful \
        $(THEOS_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/rootful
else
    _THEOS_INTERNAL_SEARCHPATHS = \
        $(TARGET_PRIVATE_FRAMEWORK_PATH) \
        $(THEOS_VENDOR_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(THEOS_PACKAGE_SCHEME) \
        $(THEOS_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(THEOS_PACKAGE_SCHEME)
endif
```

Now:

```make
ifeq ($(THEOS_PACKAGE_SCHEME),)
    _THEOS_INTERNAL_SEARCHPATHS = \
        $(TARGET_PRIVATE_FRAMEWORK_PATH) \
        $(THEOS_VENDOR_LIBRARY_PATH) \
        $(THEOS_LIBRARY_PATH) \
        $(THEOS_TARGET_LIBRARY_PATH)
else
    _THEOS_INTERNAL_SEARCHPATHS = \
        $(TARGET_PRIVATE_FRAMEWORK_PATH) \
        $(THEOS_VENDOR_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(THEOS_PACKAGE_SCHEME) \
        $(THEOS_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(THEOS_PACKAGE_SCHEME)
endif
```